recipes/laszip/all/conanfile.py

```diff
@@ -35,6 +35,8 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        if Version(self.version) >= "3.4.4":
+            self.license = "Apache-2.0"
```

Comment:
Ah, thanks. The license change is hidden in the configure method and thus less prominent than setting the class variable directly. Did not know about this option.